src/sentry/integrations/utils/sync.py

```diff
@@ -47,7 +53,7 @@ def get_user_id(projects_by_user: Mapping[int, Sequence[int]], group: Group) ->
 
 
 def sync_group_assignee_inbound(
-    integration: Integration,
+    integration: APIIntegration,
```

Comment:
Does this method need to change at all? It looks like `integration` is used a few times in this function and passed to other functions.

Reply:
It might not need to but I the models it calls are using managers that seem to be receiving APIIntegrations and I believe that's whats causing the issue details page to not render linked jira issues.

This is to mark the function as 'safe' and capable of handling only receiving an APIIntegration rather than the django object.